backend/cds/migrations/0002_auto_20220531_0905.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 3.2.8 on 2022-05-31 09:05
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("cds", "0001_initial"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="lecture",
+            name="keywords",
+            field=django.contrib.postgres.fields.ArrayField(
+                base_field=models.CharField(max_length=500),
```

Comment:
this seems an older migration, before you change back to `250`

Reply:
Good call. I make a new migration and remove this one.